pkg/tsdb/loki/api.go

```diff
@@ -324,7 +324,7 @@ func (api *LokiAPI) RawQuery(ctx context.Context, resourcePath string) (RawLokiR
 	return rawLokiResponse, nil
 }
 
-func getSupportingQueryHeaderValue(req *http.Request, supportingQueryType SupportingQueryType) string {
+func getSupportingQueryHeaderValue(supportingQueryType SupportingQueryType) string {
 	value := ""
 	switch supportingQueryType {
 	case SupportingQueryLogsVolume:
```

Comment:
Could we add comment explaining why do we have these 3 set this way (legacy reasons)

Reply:
Good idea, done in 506c4de5077b81fd6b10a126f5335974fb92c11e